packages/core/src/span.ts

```diff
@@ -0,0 +1,22 @@
+import type { SpanEnvelope, SpanItem } from '@sentry/types';
+import type { Span } from '@sentry/types/build/types/span';
+import { createEnvelope } from '@sentry/utils';
+
+/**
+ * Create envelope from Span item.
+ */
+export function createSpanEnvelope(span: Span): SpanEnvelope {
+  const headers: SpanEnvelope[0] = {
+    sent_at: new Date().toISOString(),
+  };
+
+  const item = createSpanItem(span);
+  return createEnvelope<SpanEnvelope>(headers, [item]);
+}
+
+function createSpanItem(span: Span): SpanItem {
+  const spanHeaders: SpanItem[0] = {
+    type: 'span',
+  };
+  return [spanHeaders, span];
```

Comment:
one thing - we can flush multiple spans here right? can we update this envelope to flush an array of spans? that means we can re-use it for span streaming.

Reply:
Updated here:
https://github.com/getsentry/sentry-javascript/pull/10706/commits/05c23931fe26eea2e97fad6c8aebb4ded95352fe

Let me know if this is what you had in mind!

Reply:
I thought this was set up in a way that we should have a single span item that has an array of items:

```
{}
{ type: 'span' }
[{...}, {...}]
```

Is this not what relay expects for the spans endpoint? Do we have to have to actually have multiple span items?

Reply:
Tested this with relay locally and I'm getting the following error:
```
failed to normalize span: failed to extract event payload
```

There's probably something in standalone span processing that assumes the payload will always be a single span.